src/app/component/places/places.component.scss

```diff
@@ -0,0 +1,305 @@
+@mixin set-width-height($width: 100%, $height: 100%) {
+  width: $width;
+  height: $height;
+}
+
+@mixin flex-options($flex-direction: row, $justify-content: stretch, $align-items: stretch) {
+  display: flex;
+  flex-direction: $flex-direction;
+  align-items: $align-items;
+  justify-content: $justify-content;
+}
+
+.main {
+  margin-top: -70px;
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+
+  hr {
+    margin: 21px 0 48px 0;
+    background: var(--ubs-primary-light-grey);
+  }
+
+  .places-filter {
+    @include flex-options(column, stretch, flex-start);
+
+    .filter {
+      margin-top: 26px;
+    }
+  }
+
+  .places-recommendation {
+    .recommendation-title {
+      margin: 24px 0;
+      height: 28px;
+      font-weight: bold;
+      font-size: 20px;
+      color: var(--secondary-dark-grey);
+    }
+
+    .card-container {
+      @include flex-options(row, space-between, space-between);
+
+      flex-wrap: wrap;
+    }
+  }
+
+  caption {
+    display: none;
+  }
+
+  .places-map {
+    @include flex-options(row);
+
+    .google-map {
+      height: 540px;
+      width: 100%;
+    }
+  }
+}
+
+.marker {
+  max-width: 32px;
+  max-height: 32px;
+}
+
+.close-mark::after {
+  padding: 15px;
+  display: none;
+  content: 'X';
+  position: absolute;
+  right: 0;
+  top: 0;
+  font-size: 20px;
+  color: #313131;
```

Comment:
Color  through a variable? Ask Yurii about it. Maybe that these colors will  be used.

Reply:
I'll just put another color from our existing colors because it's not from a mockup).